src/main/java/com/iri/training/repository/ConnectToBase.java

```diff
@@ -1,31 +1,22 @@
 package com.iri.training.repository;
 
-import java.sql.Connection;
-import java.sql.DriverManager;
-import java.sql.SQLException;
+import org.springframework.context.annotation.Bean;
+import org.springframework.jdbc.datasource.DriverManagerDataSource;
 
 public  class ConnectToBase {
 
-	private static Connection c;
-
-	public Connection getConnection() throws SQLException {
-
-		try {
-			Class.forName("org.sqlite.JDBC");
-			c = DriverManager.getConnection("jdbc:sqlite:db\\TrainingApp.db");
-			System.out.println("Opened database successfully");
 
+		@Bean
+		public static DriverManagerDataSource getDataSource() {
```

Comment:
I am not sure about the above, try it and if it does not work you can leave as it is